src/gwt/acesupport/snippets/snippets/r.js

```diff
@@ -152,6 +152,25 @@ var snippets = [
    {
       name: "rapply",
       content: "rapply(${1:list}, ${2:function})"
+   },
```

Comment:
I would simplify this and not worry about introducing the "!" convention. Perhaps just a single example using something like sessionInfo?